SampleApps/SPMTest/SPMTest.xcodeproj/project.pbxproj

```diff
@@ -417,7 +417,7 @@
 			isa = XCRemoteSwiftPackageReference;
 			repositoryURL = "https://github.com/braintree/braintree_ios.git";
 			requirement = {
-				branch = "cardinal-xcframework";
+				branch = master;
```

Comment:
Setting this back to master now that the `cardinal-xcframework` branch has been merged in.